interface.idl

```diff
@@ -56,4 +56,8 @@ namespace net {
     oneway tx(payload: bytes);
 }
 
+namespace time {
+    rpc readrtc(format: str) -> (currenttime: str);
```

Comment:
Returning string requires some allocations and is not efficient. Please return as integers:

```
    rpc read() -> (year: uint32, month: uint8, day: uint8, hour: uint8, minute: uint8, second: uint8, day_of_week: uint8);
```